docker/e2e/cypress/integration/migrating_users.spec.js

```diff
@@ -38,6 +40,26 @@ describe('user migration plugin', () => {
         submitCredentials(ADMIN_USERNAME, ADMIN_PASSWORD);
     }
 
+    /**
+     * Configures v1 admin theme, because User Federation plugin config does not work with v2 yet.
```

Comment:
Could you elaborate? Sounds like a serious issue.

Reply:
With keycloak v19 the default admin gui is "keycloak.v2" which unfortunately seems to be incompatible with the plugin config options as defined in `com.danielfrak.code.keycloak.providers.rest.ConfigurationProperties#PROPERTIES`:
https://id-sportsid-dev.k8s.risedev.at/admin/master/console/#/realms/master/users/d210340d-6342-4d03-ae5e-fad33a330c21/role-mappings

Reply:
With v1 which is still supported everything works fine.

Reply:
It seems the new user console is a bit of a mess, from what I'm reading... It's unfortunate that it's such a hassle to upgrade.

1. We should definitely include information on the incompatibility in the `README.md`, for example under the compatibility history we could have something like:
```markdown
## Keycloak 19+ compatibility notice

Due to the new Keycloak React admin console not yet supporting custom configuration properties, you will need to use the old admin console to configure this plugin. More details on the new admin console and how to disable it are available here:

https://www.keycloak.org/docs/latest/upgrading/index.html#new-admin-console-is-now-the-default-console
```

2. As described in the link above, I think a better solution for us would be to use `--features-disabled=admin2` in the Keycloak dockerfile (should just be a matter of adding it as part of the `ENTRYPOINT`) instead of the whole `setAdminV1Theme();` shebang.

3. An issue should be created on [Keycloak's Github](https://github.com/keycloak/keycloak/issues) to make sure that it gets noticed and fixed by them before they remove the old UI. As someone who encountered this first-hand, do you have time to do it or would you prefer if I did?

Reply:
You're right about the new admin console. And unfortunately, at least in my case it's still buggy too.
Ad 1) Agreed.
Ad 2) I'm glad its you who proposed this. Will happily do that.
Ad 3) I can do that. I should be able to spare some time today to finish this MR.

Reply:
Awesome, thanks a lot for your help with this!

It might also be a good idea to link to the created Github issue from the `README.md` notice, so that it's easier to track when using the old UI is no longer needed from our perspective.